src/test/java/calculator/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,54 @@
+package calculator;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class StringAddCalculatorTest {
+
+    @Test
+    public void splitAndSum_null_또는_빈문자() {
```

Comment:
```suggestion
    void splitAndSum_null_또는_빈문자() {
```
junit5부터는 메소드의 pulibc이 제거되어도 됩니다 (https://junit.org/junit5/docs/current/user-guide/#writing-tests-classes-and-methods)